spec/ios_get_app_version_spec.rb

```diff
@@ -5,37 +5,54 @@
     it 'parses the xcconfig file format correctly and gets the public version' do
       xcconfig_mock_content = <<~CONTENT
         // a comment
-        VERSION_SHORT=6
-        VERSION_LONG=6.30.0
+        VERSION_SHORT = 6
+        VERSION_LONG = 6.30.0
       CONTENT
 
-      allow(File).to receive(:exist?).and_return(true)
-
       expect_version(xcconfig_mock_content: xcconfig_mock_content, expected_version: '6.30')
     end
 
     it 'parses the xcconfig file format correctly and gets the public hotfix version' do
       xcconfig_mock_content = <<~CONTENT
-        VERSION_SHORT=6
+        VERSION_SHORT = 6
         // a comment
-        VERSION_LONG=6.30.1
+        VERSION_LONG = 6.30.1
```

Comment:
Interestingly, that means that now we do have tests for the cases where there are spaces around the key and value… but we also don't have tests for cases without spaces around the `=` anymore. Could it be worth duplicating those 2 tests but have one with spaces and one without?

Reply:
My idea here was that the spaces would be where parsing would fail. But you're right, it's probably worth it adding this test as well as some sort of invalid format.

Reply:
Implemented here: https://github.com/iangmaia/release-toolkit/pull/1/commits/d72c9366bc8d28da8113808ca06579885f90fe91